Micromouse/Mouse.cpp

```diff
@@ -178,10 +174,33 @@ void Mouse::followPath(bool useCaution) {
 
 unsigned Mouse::move(DirectionVector movement, bool keepGoing,
                      bool useCaution) {
-    (void)movement;
+#if !defined(__MK66FX1M0__) && !defined(__MK20DX256__)
```

Comment:
At some point I would like to see the PC only code moved farther down the chain (ei into `Hardware::moveForward` and `rotate`). Doesn't need to be now.